manager/manager.go

```diff
@@ -101,6 +101,9 @@ type Options struct {
 	// If the list is empty, all probes will be activated.
 	ActivatedProbes []string
 
+	// ExcludedProbes is a list of probes that should not even be verified.
+	ExcludedProbes []string
```

Comment:
A tiny suggestion, but thoughts on `DisabledProbes` as the name? It feels like it might better describe the difference in usage versus `ActivatedProbes`.

Reply:
The problem is there are 3 states a probe program can be in:
1. Exists only in the ELF file
2. Verified, but not attached/activated
3. Attached/Activated

For me, disabled somewhat implies being in state 2. In that, you could enable it later? Which is not the case for programs which have been "excluded".